### PR TITLE
Make usage field passthrough

### PR DESCRIPTION
**Pull request by ycombinator**
With `xpack.enabled: true` set in the `kibana` module, the `stats` metricset will ship Kibana stats into `.monitoring-kibana-*` X-Pack Monitoring indices. One field that's part of Kibana stats going into `.monitoring-kibana-*` is the `usage` field. It contains telemetry information exposed by Kibana, including various plugins in Kibana.

Kibana plugin authors may wish to include arbitrary structured data under this `usage` field. Currently, whenever the structure of this data changes (new fields are introduced, etc.), the [schema for the `usage` field](https://github.com/elastic/beats/blob/master/metricbeat/module/kibana/stats/data_xpack.go#L74-L107) in in the `kibana/stats` (x-pack) metricset must be changed as well.

Clearly this is not a scalable solution. This PR addresses this coupling by making the `usage` field a passthrough. That is, whatever structured data is under the `usage` field in the response returned by the Kibana `GET /api/stats?extended=true` API will be indexed into `.monitoring-kibana-*` indices as-is.

### CODE REVIEW

File: metricbeat/module/kibana/stats/data_xpack.go [hunk 1]
```diff
@@ -71,40 +71,40 @@ var (
...
-					"_all":          c.Int("all"),
-					"csv":           reportingCsvDict,
-					"printable_pdf": reportingPrintablePdfDict,
-					"status":        reportingStatusDict,
-					"lastDay":       c.Dict("last_day", reportingPeriodSchema, c.DictOptional),
-					"last7Days":     c.Dict("last_7_days", reportingPeriodSchema, c.DictOptional),
-				}, c.DictOptional),
-			},
-		}),
+		// "usage": c.Dict("usage", s.Schema{
```
Comment (ruflin) [inline metricbeat/module/kibana/stats/data_xpack.go]: Leftover?

Comment (ycombinator) [reply metricbeat/module/kibana/stats/data_xpack.go]: Yeah, oops! Just fixed in 3dc1365.

**Comment by ycombinator** [comment]
@chrisronline When you get a chance, could you test this PR functionally with Kibana please?

**Comment by ycombinator** [comment]
jenkins, test this

**Comment by ycombinator** [comment]
jenkins, test this

**Comment by ycombinator** [comment]
Thanks @chrisronline. @ruflin This is ready for you to re-review/merge whenever you get a chance. Thanks.

**Pull request merged**